src/components/TodoModal/TodoModal.tsx

```
@@ -1,48 +1,72 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+import { TodoModalList } from './TodoModalList';
+import { getUser } from '../../api';
+
+interface PropsUser {
+  todos: Todo[];
+  isModalClosed: (event: boolean) => void;
+  clickedValue: number;
+}
+
+export const TodoModal: React.FC<PropsUser> = (
+  { todos, isModalClosed, clickedValue },
+) => {
+  const [users, setUsers] = useState<User[]>([]);
+  const [combineData, setCombineData] = useState<User[]>([]);
+
+  useEffect(() => {
+    let isMounted = true;
+
+    const fetchData = async () => {
+      try {
+        const userData = await getUser(clickedValue);
+
+        if (isMounted) {
+          setUsers([userData]);
+        }
+      } catch (error) {
+        console.warn(error); // eslint-disable-line no-console
+      }
```

lets throw an error instead of console log